src/test/java/teammates/sqlui/webapi/GetInstructorPrivilegeActionTest.java

```diff
@@ -0,0 +1,231 @@
+package teammates.sqlui.webapi;
+
+import static org.mockito.Mockito.when;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.InstructorPermissionRole;
+import teammates.common.datatransfer.InstructorPermissionSet;
+import teammates.common.datatransfer.InstructorPrivileges;
+import teammates.common.util.Const;
+import teammates.storage.sqlentity.Course;
+import teammates.storage.sqlentity.Instructor;
+import teammates.ui.output.InstructorPrivilegeData;
+import teammates.ui.webapi.EntityNotFoundException;
+import teammates.ui.webapi.GetInstructorPrivilegeAction;
+
+/**
+ * SUT: {@link GetInstructorPrivilegeAction}.
+ */
+public class GetInstructorPrivilegeActionTest extends BaseActionTest<GetInstructorPrivilegeAction> {
+    private static String feedbackSessionName = "test-feedback-session";
+    private static String studentSectionName = "test-section";
+    private Instructor testInstructor1OfCourse1;
+    private Instructor testInstructor2OfCourse1;
+
+    @Override
+    String getActionUri() {
+        return Const.ResourceURIs.INSTRUCTOR_PRIVILEGE;
+    }
+
+    @Override
+    String getRequestMethod() {
+        return GET;
+    }
+
+    @BeforeMethod
+    void prepareTestData() {
```

Comment:
nit: to standardize with other (i think the majority of) tests, can we call this setUp()